how-to.md

```diff
@@ -1,3 +1,13 @@
+## Getting access to Simeon
+
+If you don't yet have a Simeon instance and you want to start a Simeon trial, please go to [Simeoncloud.com](https://www.simeoncloud.com/) to request a demo. If you decide to do a trial after the demo or if you purchased a subscription with us, you will be able to book an onboarding meeting with one of our engineers who will create your Simeon instance for you (no need to create a DevOps org yourself, unless you want to provide your own Azure DevOps organization). You will be given access to your Simeon instance during your onboarding meeting.
```

Comment:
Not at all a big deal, but in the same sentence we say "DevOps org" and "Azure DevOps organization". might be preferable to use the same terminology in both places

Reply:
resolved